examples/quickrefl/core/model/realdatamodel.cpp

```diff
@@ -96,6 +97,30 @@ void RealDataModel::addDataToGroup(DataGroupItem* data_group, RealDataStruct& da
     graph->setDataItem(data);
 }
 
+//! Insert the data into the group item
+void RealDataModel::removeAllDataFromNode(DataCollectionItem* data_node)
```

Comment:
Please provide consistent naming by changing it, to `removeAllDataFromCollection`. I think it is better to get rid from `Node` everywhere in the favor of `Collection`. Also please rename file `datasetitem.h` to `dataitems.h` to make it consistent with the rest of the project.